.github/workflows/macos-sanity.yaml

```diff
@@ -0,0 +1,39 @@
+name: macOS sanity checks
+on:
+  # Only run on pull requests: not pushes
+  pull_request:
+    branches: [ "master", "release/**" ]
+
+jobs:
+  cla-check:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "1.10.x"
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Install squashfs from homebrew
+        uses: artemnovichkov/action-homebrew@master
+        args: "install squashfs"
```

Comment:
@zyga @stolowski should we do that manually instead? What's the right set of commands here?

Reply:
I don't use homebrew but IIRC the commands are rather simple:
```
brew install squashfs
```

There's some nice documentation on https://brew.sh/index_pl

Reply:
`brew install squashfs` pulls dependencies for squashfs: lz4, lzo and zstd; I'll try to verify if this is sufficient.